src/app/modules/main/app_search/module.nim

```diff
@@ -146,80 +162,85 @@ method searchMessages*(self: Module, searchTerm: string) =
-  let channelGroups = self.controller.getChannelGroups()
+  # Add Personal section chats
+  let myPubKey = singletonInstance.userProfile.getPubKey()
+  let personalItems = self.getResultItemFromChats(myPubKey, self.controller.getChatsForPersonalSection())
+  var channels = personalItems
+
+  # Add Communities
+  let communities = self.controller.getJoinedCommunities()
+
   let searchTerm = self.controller.searchTerm().toLower
-  for channelGroup in channelGroups:
-    let isCommunity = channelGroup.channelGroupType == ChannelGroupType.Community
-    if(self.controller.searchLocation().len == 0 and
-        channelGroup.name.toLower.contains(searchTerm)):
+  for community in communities:
+    if self.controller.searchLocation().len == 0 and
+        community.name.toLower.contains(searchTerm):
       let item = result_item.initItem(
-        channelGroup.id,
+        community.id,
         content="",
         time="",
-        titleId=channelGroup.id,
-        title=channelGroup.name,
-        if (isCommunity):
-            SEARCH_RESULT_COMMUNITIES_SECTION_NAME
-          else:
-            SEARCH_RESULT_CHATS_SECTION_NAME,
-        channelGroup.images.thumbnail,
-        channelGroup.color,
+        titleId=community.id,
+        title=community.name,
+        SEARCH_RESULT_COMMUNITIES_SECTION_NAME,
+        community.images.thumbnail,
+        community.color,
         badgePrimaryText="",
         badgeSecondaryText="",
-        channelGroup.images.thumbnail,
-        channelGroup.color,
+        community.images.thumbnail,
+        community.color,
         badgeIsLetterIdenticon=false)
 
-      self.controller.addResultItemDetails(channelGroup.id, channelGroup.id)
+      self.controller.addResultItemDetails(community.id, community.id)
       items.add(item)
 
     # Add channels
-    if((self.controller.searchSubLocation().len == 0 and self.controller.searchLocation().len == 0) or
-        self.controller.searchLocation() == channelGroup.id):
-      for chatDto in channelGroup.chats:
-        var chatName = chatDto.name
-        var chatImage = chatDto.icon
-        var colorHash: ColorHashDto = @[]
-        var colorId: int = 0
-        let isOneToOneChat = chatDto.chatType == ChatType.OneToOne
-        if(isOneToOneChat):
-          (chatName, chatImage) = self.controller.getOneToOneChatNameAndImage(chatDto.id)
-          colorHash = self.controller.getColorHash(chatDto.id)
-          colorId = self.controller.getColorId(chatDto.id)
-
-        var rawChatName = chatName
-        if(chatName.startsWith("@")):
-          rawChatName = chatName[1 ..^ 1]
-
-        if(rawChatName.toLower.contains(searchTerm)):
-          let item = result_item.initItem(
-            chatDto.id,
-            content="",
-            time="",
-            titleId=chatDto.id,
-            title=chatName,
-            if isCommunity:
-                SEARCH_RESULT_CHANNELS_SECTION_NAME
-              else:
-                SEARCH_RESULT_CHATS_SECTION_NAME,
```

Comment:
Nice catch. I removed all the code that used `isCommunity` but missed the difference here. It was not a huge deal since it was just a string difference, but I fixed it.